aws-s3-controller.yaml

```
@@ -0,0 +1,50 @@
+package:
+  name: aws-s3-controller
+  version: 1.0.21
+  epoch: 0
+  description: ACK service controller for Amazon Simple Storage Service (S3)
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 2be16650fa025711d45f3f8042904e6ff0c211fd
+      repository: https://github.com/aws-controllers-k8s/s3-controller.git
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -X main.version=v${{package.version}}
+        -X main.buildHash=$(git rev-parse HEAD)
+        -X main.buildDate=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
+      packages: ./cmd/controller
+      output: controller
+
+  - runs: |
+      mkdir -p ${{targets.contextdir}}
+      cp LICENSE ${{targets.contextdir}}/
+      cp ATTRIBUTION.md ${{targets.contextdir}}/
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: "Compatibility package to place binaries in the location expected by aws-s3-controller"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/bin
+          ln -sf /usr/bin/controller ${{targets.contextdir}}/bin/controller
+    test:
+      pipeline:
+        - runs: "[ \"$(stat -c '%N' /bin/controller 2>/dev/null)\" = \"'/bin/controller' -> '/usr/bin/controller'\" ]"
+
+update:
+  enabled: true
+  github:
+    identifier: aws-controllers-k8s/s3-controller
+    strip-prefix: v
+    use-tag: true
+
+test:
+  pipeline:
+    - runs: stat /usr/bin/controller
```

I think we can consider adding the following in here to start with. This way we're verifying the binary invocation is working. 
```suggestion
    - runs: |
        stat /usr/bin/controller
        controller --help 2>&1 | grep webhook
        controller --aws-region us-east-1 2>&1 | grep -E 'session|token'
```